.travis.yml

```diff
@@ -297,7 +284,7 @@ install:
               echo "$COMPONENTS" | xargs -n1 -I{} tar --append -f ~/php-ext/composer-lowest.lock.tar {}/composer.lock
           elif [[ $PHP = hhvm* ]]; then
               rm src/Symfony/Bridge/PhpUnit -Rf
-              $PHPUNIT --exclude-group no-hhvm,benchmark,intl-data
+              $PHPUNIT --exclude-group no-hhvm,benchmark,intl-data,redis,memcached
```

Comment:
I don't think we need to skip them. They should skip themselves automatically anyway. Not skipping here would ensure we added the proper checks in the test suite.

Reply:
I'll double check, but they might be skipped based on the extension being present, not the service being available.

Reply:
I made sure tests are skipped properly.